### PR TITLE
[Wav2Vec2, Hubert] Fix ctc loss test

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

This PR fixes the `ctc_loss` test. The problem is that PyTorch ctc_loss implementation https://pytorch.org/docs/stable/generated/torch.nn.CTCLoss.html seems to use too many approximation to ensure deterministic behavior for a test. So in the PR I make the test much less aggressive by simply checking that the loss can be computed.

We've trained 250 models during the Wav2Vec2 sprint using this function so I think we can be quite certain that it works as expected. 

Not an ideal solution, but I don't really see how to make this "sum == mean" test really work since it's so flaky

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Thanks @patrickvonplaten
